src/_P092_DLbus.ino

```diff
@@ -337,9 +267,10 @@ boolean Plugin_092(uint8_t function, struct EventStruct *event, String& string)
         // pin number is changed -> run a new init
         P092_init = false;
 
-        if (DLbus_Data->ISR_DLB_Pin != 0xFF) {
+        if (P092_data->DLbus_Data->IsISRset) {
           // interrupt was already attached to P092_DLB_Pin
-          detachInterrupt(digitalPinToInterrupt(DLbus_Data->ISR_DLB_Pin));
+          detachInterrupt(digitalPinToInterrupt(P092_data->DLbus_Data->ISR_DLB_Pin));
+          P092_data->DLbus_Data->IsISRset = false;  //to ensure that a new interrupt is attached in P092_data->init()
```

Comment:
I'm not sure whether this is the optimal place to set the IsISRset to false.

I'm not that familiar with timings of esp easy but since the new interrupt is attached in PLUGIN_ONCE_A_SECOND i think under bad circumstances PLUGIN_ONCE_A_SECOND could be called before PLUGIN_INIT (the place where the new pin is set) and then the interrupt is not attached to the correct pin.

Therefore i suggested to place setting IsISRset to false before setting the new pin during initialization (where you already added it ;) )

Reply:
Yep, I think that's a valid point.
Reset the flag first before the detachInterrupt.
And maybe also place it in a separate scope so the compiler may not perform an out-of-order optimization.
```c++
{
  P092_data->DLbus_Data->IsISRset = false;  //to ensure that a new interrupt is attached in P092_data->init()
}
detachInterrupt(digitalPinToInterrupt(P092_data->DLbus_Data->ISR_DLB_Pin));
```

Reply:
i would completely delete it here because its already done in init (triggered by flag P092_init).

Reply:
Hmm for that I may have to look a bit more into it...